docpages/example_programs/music_and_audio/oggopus.md

```diff
@@ -115,11 +111,11 @@ int main(int argc, char const *argv[])
 
 			/* Now we ensure the encoding is correct for Discord */
 			if (header.channel_count != 2 && header.input_sample_rate != 48000) {
-				fprintf(stderr,"Wrong encoding for Discord, must be 48000Hz sample rate with 2 channels.\n");
+				fprintf(stderr,"Wrong encoding for Discord, must be 48000 Hz sample rate with 2 channels.\n");
```

Comment:
Conventional notation here would indeed be 48000Hz

Reply:
Kind of. You are recommended to use the space but both are accepted.